tests/proofs/lqt/lqt-spec.md

```diff
@@ -122,7 +122,7 @@ module LQT-TOKEN-MINTORBURN-SPEC
      andBool Sender ==K Admin
      andBool #tokensFor(Tokens, Address) +Int Quantity >Int 0
 
-   claim <k> #runProof(MintOrBurnParams(Quantity, Address)) => Aborted(?_, ?_, ?_, ?_) ... </k>
+   claim <k> #runProof(MintOrBurnParams(Quantity, Address)) => Aborted(?_, ?_, ?_, ?_) </k>
```

Comment:
@sskeirik does this affect the generality of the proofs? I would like to have them be true in _any_ execution context, but for some languages that what ending in `.K` means anyway.